src/libmongoc/tests/test-mongoc-uri.c

```diff
@@ -2416,10 +2268,10 @@ test_mongoc_uri_duplicates (void)
 
    RECREATE_URI (MONGOC_URI_READPREFERENCE
                  "=secondary&" MONGOC_URI_MAXSTALENESSSECONDS
-                 "=1&" MONGOC_URI_MAXSTALENESSSECONDS "=2");
+                 "=100&" MONGOC_URI_MAXSTALENESSSECONDS "=200");
```

Comment:
Was this intentional?

Reply:
Adding in additional tests revealed that we handle maxStalenessSeconds correctly:

From the MDB docs:
"You must specify a maxStalenessSeconds value of 90 seconds or longer: specifying a smaller maxStalenessSeconds value will raise an error."

From the C driver URI docs:
"The smallest allowed value for maxStalenessSeconds is 90 seconds."

Even though we aren't actually enforcing this lower limit at the time when we set this parameter, I thought it was silly to be using numbers <90 to test our "success" case.  We should probably fix this, but I didn't want to add that fix to this code review.